sdk/purview/azure-purview-workflow/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 1.0.0b2 (2023-10-16)
+
+### Features Added
+
+- Add `/workflows/{id}/validate` API.
+- Add admin query options to workflow / workflow run APIs.
+
+### Breaking Changes
+
+- Split `PurviewWorkflowClient` into `UserRequestsClient`, `WorkflowClient`, `WorkflowRunClient` and `WorkflowTaskClient`.
```

Comment:
```suggestion
- Add Operation Group `XXX`.
```
In Python, we call it Operation Group instead of sub client.

Reply:
Updated changlog, thanks.